libsolidity/AST.h

```diff
@@ -574,6 +581,7 @@ class EventDefinition: public Declaration, public VariableScope, public Document
 
 private:
 	ASTPointer<ParameterList> m_parameters;
+	bool m_anonymous;
```

Comment:
For clarity and safty you can init the field in declaration: `bool m_anonymous = false`.